modules/nf-core/rtgtools/rocplot/tests/main.nf.test

```diff
@@ -0,0 +1,69 @@
+nextflow_process {
+
+    name "Test Process RTGTOOLS_ROCPLOT"
+    script "../main.nf"
+    process "RTGTOOLS_ROCPLOT"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "rtgtools"
+    tag "rtgtools/rocplot"
+
+    test("homo_sapiens - tsv") {
+
+        setup {
+            run("UNTAR") {
+                script "../../../untar/main.nf"
+                process {
+                    """
+                    input[0] = [
+                        [ id:'test' ],
+                        file(params.test_data['homo_sapiens']['genome']['genome_21_sdf'])
```

Comment:
Is there a reason to skip `checkIfExists` here but not below?

Reply:
Not really, I just copy pasted from the pytest files. But good idea to add it!

Reply:
It was intentional it seems. Somehow the check doesn't work even though the file exists... weird